content/docs/practices/instrumentation.md

```diff
@@ -32,6 +32,9 @@ this category.
 The key metrics in such a system are the number of performed queries, errors,
 and latency. The number of in-progress requests can also be useful.
 
+When exposing operation counters, create seperate metrics for total and failed number of
+requests. That makes calculating percental error rates easier.
```

Comment:
Now I found it. Is Failures as Subsystem? Maybe that can be improved.
And maybe it should be made more explicit. As I said, I still get confused why not use labels but separate metrics and the docs don't really clarify that (to be fair, neither does my change.)